en/docs/chapter_sorting/quick_sort.md

```diff
@@ -35,11 +35,11 @@ The core operation of quick sort is "pivot partitioning," aiming to: select an e
 === "<9>"
     ![pivot_division_step9](quick_sort.assets/pivot_division_step9.png)
 
-After the pivot partitioning, the original array is divided into three parts: left sub-array, pivot, and right sub-array, satisfying "any element in the left sub-array $\leq$ pivot $\leq$ any element in the right sub-array." Therefore, we only need to sort these two sub-arrays next.
+After the pivot partitioning, the original array is divided into three parts: left subarray, pivot, and right subarray, satisfying "any element in the left subarray $\leq$ pivot $\leq$ any element in the right subarray." Therefore, we only need to sort these two subarrays next.
```

Comment:
"Therefore, we only need to sort these two subarrays next." => Therefore, we then only need to sort these two subarrays.